src/biogeophys/LunaMod.F90

```diff
@@ -1250,8 +1256,8 @@ subroutine NUE(O2a, ci, tgrow, tleaf, NUEj,NUEc,Kj2Kc)
   real(r8) :: awc                                     !second deminator term for rubsico limited carboxylation rate based on Farquhar model
   real(r8) :: c_p                                     !CO2 compenstation point (Pa)
   
-  Fc = VcmxTKattge(tgrow, tleaf) * Fc25
-  Fj = JmxTKattge(tgrow, tleaf) * Fj25
+  Fc = VcmxTLenuning(tgrow, tleaf) * Fc25
```

Comment:
There is a typo here: Lenuning should be Leuning. Have you confirmed that the code you pushed compiles and runs?